### PR TITLE
[Fixture] Extend fixtures

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

Screenshot 2020-01-26 at 18 47 46
Screenshot 2020-01-26 at 18 47 59
Screenshot 2020-01-26 at 18 48 07
Screenshot 2020-01-26 at 18 48 18

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml [hunk 1]
```diff
@@ -9,21 +9,62 @@ sylius_fixtures:
...
+                            - 'ES'
+                            - 'CN'
+                            - 'UK'
                         zones:
                             US:
                                 name: 'United States of America'
                                 countries:
                                     - 'US'
+                            WORLD:
+                                name: 'World'
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml]:
```suggestion
                                name: 'Foreign countries'
```
or
```suggestion
                                name: 'Foreign zone'
```

I suppose someone may expect more countries in the world ;)

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml [hunk 2]
```diff
@@ -9,21 +9,62 @@ sylius_fixtures:
...
                     options:
                         currencies:
                             - 'USD'
+                            - 'PLN'
+                            - 'CAD'
+                            - 'CNY'
+                            - 'NZD'
+                            - 'GBP'
+                            - 'AUD'
+                            - 'MXN'
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml]: What about `EUR`?

**Comment by lchrusciel** [comment]
Thank you, Adam! :1st_place_medal:

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml (hunk 1 above)
Comment (AdamKasp) [reply src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml]: 'World' is a value taken from task specification, so for now, I leave this name. but I think your suggestion is ok :)

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml (hunk 2 above)
Comment (oallain) [reply src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml]: Yes, FR, DE, ES ... use Euro :thinking:

Comment (lchrusciel) [reply src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml]: Done in #11070 :)

**Pull request merged**